src/browser/convert_options.js

```diff
@@ -262,8 +262,10 @@ module.exports = {
             newOptions.permissions = options.permissions;
         }
 
-        if (options.hasOwnProperty('preload')) {
-            newOptions.preload = options.preload;
+        if (typeof options.preload === 'string') {
+            newOptions.preload = [{ url: options.preload }];
```

Comment:
@Roma, I originally had this here, but had to "denormalize" to [preload_fetch.js](https://github.com/HadoukenIO/core/blob/develop/src/browser/preload_scripts.ts#L67) and [api_decorator.js](https://github.com/HadoukenIO/core/blob/develop/src/renderer/api-decorator.js#L522) it because this code path was not happening (or was not happening early enough) for `preload` coming from manifest. Did you test manifest use case? If it fails, try moving this code to where you expected an array.